### ISSUE TITLE
CG: simplify name

### ISSUE THREAD

**Issue by jernst** [swicg/potential-charters#18]
Is it possible to shorten the name of the "W3C Social Web Incubator Community Group" and at least remove the "incubator" from it? IMHO the incubation happened a long time ago...

Ideally we would migrate to SocialCG as the handle and to be consistent, the group could be the "Social Community Group".

I'd really like to see the "swicg" tongue-breaker disappear, and even more so the several handles we use to refer to the same thing. And then make Github orgs etc consistent.

**Comment by bumblefudge** [comment]
I defer to @dmitrizagidulin on how and when, but I _would_ support this namechange and maybe the ratification of the CG charter would be a good occasion for this update?

**Comment by dmitrizagidulin** [comment]
@jernst I agree with you that the 'I' in 'swicg' is unnecessary at this point.

The main reason to keep it unchanged is the `public-swicg@w3.org` mailing list (I'm not sure what affordances we have for renaming mailing list addresses).

(Incidentally, the 'i' is often added anyways when you try to pronounce the `swcg` abbreviation).

**Comment by evanp** [comment]
I love the name SocialCG, which is I think the name we selected a long time ago:

> Accept majority of straw poll as SocialCG for group shortname.

https://github.com/swicg/meetings/blob/main/RESOLUTIONS.md
